dolphinscheduler-api-test/dolphinscheduler-api-test-case/src/test/java/org/apache/dolphinscheduler/api/test/entity/HttpResponse.java

```diff
@@ -29,4 +32,12 @@ public class HttpResponse {
     private int statusCode;
 
     private HttpResponseBody body;
+
+    private Map<String, String> headers = new HashMap<>();
+
+    public HttpResponse(int statusCode, HttpResponseBody body) {
+        this.statusCode = statusCode;
+        this.body = body;
+        this.headers = new HashMap<>();
+    }
```

Comment:
Why we need this sinice we already hava `@AllArgsConstructor` and `private Map<String, String> headers = new HashMap<>();`

Reply:
It would be better to add `@Builder` if we hope to add many constructors.